### DISCUSSION TITLE
Crash 0xc0000409

### DISCUSSION THREAD

**Discussion by norrisgc** [WinMerge/winmerge#1543] [Q&A] [answered]
Just started using WinMerge, so not entirely sure what I need to report, but this is what is in the event log:

Faulting application name: WinMergeU.exe, version: 2.16.24.0, time stamp: 0x63594b61
Faulting module name: WinMergeU.exe, version: 2.16.24.0, time stamp: 0x63594b61
Exception code: 0xc0000409
Fault offset: 0x002533c2
Faulting process ID: 0x2ca0
Faulting application start time: 0x01d8eb7843ee030e
Faulting application path: C:\Program Files (x86)\WinMerge\WinMergeU.exe
Faulting module path: C:\Program Files (x86)\WinMerge\WinMergeU.exe
Report ID: eff1a79b-a5a8-4354-b559-9c2ba4a5a0e6
Faulting package full name: 
Faulting package-relative application ID: 

I had a three-way network drive compare open, and I started a binary compare on two matching files (i.e. only present on 2 of the three drives) and WinMerge almost immediately quit. (This is repeatable.)

**Comment by Gitoffthelawn** [comment, marked as answer]
Probably should be an issue report.  Please specify OS version.  Thanks.

Issues:
https://github.com/WinMerge/winmerge/issues

**Reply by norrisgc** [reply]
Done: [https://github.com/WinMerge/winmerge/issues/1544](url)

Thanks

**Comment by Gitoffthelawn** [comment]
Just for reference, the issue link is #1544
